### PR TITLE
[INLONG-537][Sort][Agent] Add TubeMQ connector docs and update Agent docs

### LINKED ISSUE #537: [Feature][Sort][Agent] Add TubeMQ connector docs and update Agent docs
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by GanfengTan**
Add TubeMQ connector docs and update Agent docs.

- Fixes #537 

### Motivation
No message

### Modifications

No message

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by healchow — APPROVED** [review]

File: docs/data_node/extract_node/tube.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: TubeMQ
+sidebar_position: 11
+---
+
+import {siteVariables} from '../../version';
```
Comment (thesumery) [inline docs/data_node/extract_node/tube.md:6]: image

Comment (thesumery) [reply docs/data_node/extract_node/tube.md:6]: what does it means?

File: docs/data_node/extract_node/tube.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Dependencies  
+
+In order to set up the `TubeMQ Extract Node`, the following provides dependency information for both projects using a
+build automation tool (such as Maven or SBT) and SQL Client with Sort Connectors JAR bundles.
+
+### Maven dependency
+
+<pre><code parentName="pre">
+{`<dependency>
```
Comment (thesumery) [inline docs/data_node/extract_node/tube.md:26]: image
wrong format

File: docs/data_node/extract_node/tube.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Overview
+
+[Apache InLong TubeMQ](https://inlong.apache.org/docs/modules/tubemq/overview) is a distributed, open source pub-sub messaging and steaming platform for real-time workloads, trillions of massive data precipitation.
+
+## Version
+
+| Extract Node          | Version                                                      |
+| --------------------- | ------------------------------------------------------------ |
+| [TubeMQ](./tube.md) | [TubeMQ](https://inlong.apache.org/docs/next/modules/tubemq/overview): >=0.1.0<br/> |
```
Comment (thesumery) [inline docs/data_node/extract_node/tube.md:16]: Extra `:`

File: docs/data_node/extract_node/tube.md (hunk 2 above)
Comment (thesumery) [reply docs/data_node/extract_node/tube.md:26]: Please correct the above format errors  in the Chinese document in the same time.

File: docs/data_node/extract_node/tube.md (hunk 3 above)
Comment (thesumery) [reply docs/data_node/extract_node/tube.md:16]: Please correct the above format errors  in the Chinese document in the same time.

File: docs/modules/agent/overview.md [hunk 4]
```diff
@@ -25,6 +25,21 @@ User-configured path monitoring, able to monitor the created file information
 Directory regular filtering, support YYYYMMDD+regular expression path configuration
 Breakpoint retransmission, when InLong-Agent restarts, it can automatically re-read from the last read position to ensure no reread or missed reading.
 
+#### File options
+| Parameter                     | Required | Default value | Type   | Description                                                  |
+| ----------------------------- | -------- | ------------- | ------ | ------------------------------------------------------------ |
+| pattern                       | required | (none)        | String | File pattern. For example: /root/[*].log      |
+| timeOffset                    | optional | (none)        | String | File name includes time, for example: *** YYYYMMDDHH *** YYYY represents the year, MM represents the month, DD represents the day, and HH represents the hour, *** is any character. '1m' means one minute after, '-1m' means one minute before. '1h' means one hour after, '-1h' means one hour before. '1d' means one day after, '-1d' means one day before.|
```
Comment (thesumery) [inline docs/modules/agent/overview.md:32]: *** YYYYMMDDHH ***  should be ` YYYYMMDDHH `  ？

Comment (thesumery) [reply docs/modules/agent/overview.md:32]: Please correct the above format errors in the Chinese document in the same time.

File: docs/modules/sort/quick_start.md [hunk 5]
```diff
@@ -14,7 +14,8 @@ Currently, InLong Sort relies on Flink-1.13.5. Chose `flink-1.13.5-bin-scala_2.1
 - InLong Sort file, [Download](https://inlong.apache.org/download/) `apache-inlong-[version]-bin.tar.gz`
 - Data Nodes Connectors, [Download](https://inlong.apache.org/download/) `apache-inlong-[version]-sort-connectors.tar.gz`
 
-Notice: Please put required Connectors jars into under `FLINK_HOME/lib/` after download.
+Notice: Please put required Connectors jars into under `FLINK_HOME/lib/` after download.  
+Put mysql-connector-java:8.0.21 jar to `FLINK_HOME/lib/` when you use `mysql-cdc-inlong` connector.
```
Comment (dockerzhang) [inline docs/modules/sort/quick_start.md:18]: please use [mysql-connector-java-8.0.27.jar](https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.27/mysql-connector-java-8.0.27.jar) and add the link.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort/quick_start.md [hunk 6]
```diff
@@ -12,7 +12,8 @@ sidebar_position: 2
 - InLong Sort 运行文件，[下载](https://inlong.apache.org/zh-CN/download/) `apache-inlong-[version]-bin.tar.gz`
 - 数据节点 Connectors，[下载](https://inlong.apache.org/zh-CN/download/) `apache-inlong-[version]-sort-connectors.tar.gz`
 
-注意：Connectors 下载后可以将需要的 jars 放到`FLINK_HOME/lib/`下。
+注意：Connectors 下载后可以将需要的 jars 放到`FLINK_HOME/lib/`下。  
+如果使用`mysql-cdc-inlong` 连接器，请将 `mysql-connector-java:8.0.21` jar 包放到 `FLINK_HOME/lib/`下。
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/sort/quick_start.md:16]: ditto

**Review by thesumery — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**